pom.xml

```diff
@@ -165,6 +166,20 @@
       <artifactId>angular-chart.js</artifactId>
       <version>${webjars.bower.angular-chartjs.version}</version>
     </dependency>
+
+    <!-- testing related dependencies -->
+    <dependency>
+      <groupId>io.github.benas</groupId>
+      <artifactId>random-beans</artifactId>
+      <version>${randombeans.version}</version>
+    </dependency>
+    <dependency>
+      <groupId>org.mockito</groupId>

```

Comment:
Please double check this dependency. As far as I know, I don't think it is needed. Spring-boot-starter-test consists of these.